### ISSUE TITLE
flatpickr.js 12-hour Time formatting is incorrectly set

### ISSUE THREAD

**Issue by andypullen** [verbb/formie#933]
**Description**
The time values saved for the dateFormat are incorrect
![Screen Shot 2022-06-08 at 16 33 59@2x](https://user-images.githubusercontent.com/9940032/172734745-2e842b38-96fc-46a2-a5ff-94236520c829.jpg)

**Steps to reproduce**
1. Create a form with a date / time field using the flatpicker.js calendar
2. Make sure the Time field is included. 
3. Set the time format to one of the 12-hour time formats

### Form settings
Please pick from the options below.

- Multi-page form: Either
- Submission Method: Either
- Client-side Validation: Either
- Custom Form Templates: No

**Additional info**
- Plugin version: latest
- Craft version: 3.7.43
- Multi-site: Either

**Additional context**
The options for the select field are:
```
  <option value="H:i:s">23:59:59 (HH:M:S)</option>
  <option value="H:i:s A">03:59:59 PM (H:M:S AM/PM)</option>
  <option value="H:i">23:59 (HH:M)</option>
  <option value="H:i A">03:59 PM (H:M AM/PM)</option>
  <option value="i:s">59:59 (M:S)</option>
```
but according to https://flatpickr.js.org/formatting/ it should be:
```
  <option value="H:i:s">23:59:59 (HH:M:S)</option>
  <option value="h:i:s K">03:59:59 PM (H:M:S AM/PM)</option>
  <option value="H:i">23:59 (HH:M)</option>
  <option value="h:i K">03:59 PM (H:M AM/PM)</option>
  <option value="i:s">59:59 (M:S)</option>
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.0](https://github.com/verbb/formie/releases/tag/1.6.0)
